chaoscenter/authentication/pkg/entities/project.go

```diff
@@ -71,6 +75,50 @@ type ListInvitationResponse struct {
 	InvitationRole MemberRole `json:"invitationRole"`
 }
 
+type ListProjectResponse struct {
+	Projects              []*Project `json:"projects"`
+	TotalNumberOfProjects *int64     `json:"totalNumberOfProjects"`
```

Comment:
Both the fields can be made mandatory, since these fields will always be returned in the response. Empty array can be returned in projects in case no projects are found